### ISSUE TITLE
Digital Wallet Payments do not work in combination with our OfferManagement Plugin

### ISSUE THREAD

**Issue by felixbrucker** [pickware/ShopwareStripePayment#80]
Currently we try to find the order submit button like this:
https://github.com/pickware/ShopwareStripePayment/blob/5bb16df0bf18a44f47fd97dc1d786bae28d63cb3/Views/frontend/_public/src/javascript/stripe_payment_digital_wallets.js#L251

However the OfferManagement plugin adds its own submit button inside a form into the `main--actions` div and thus this button is used instead of the order submit button as it is inserted before the order submit button.

To fix this we should only select the submit button directly inside the `main--actions` div, like so:
```js
$('.confirm--content .main--actions > button[type="submit"]')
```

**Comment by svenmuennich** [comment]
Fix was released as part of v5.3.2.

**Comment by erikforss** [comment]
Since no customer is mentioned in this issue I will remove it from the board.
If somebody should be informed, please give us a hint @felixbrucker :)
